Resources/Prototypes/Entities/Mobs/Species/arachnid.yml

```
@@ -12,7 +12,7 @@
     species: Arachnid
   - type: Hunger
     baseDecayRate: 0.0125 # Spiders have slow metabolisms all things considered, so I decided to just make their hunger drain slower.
-  #- type: Carriable # Carrying system from nyanotrasen.
+  - type: Carriable # Carrying system from nyanotrasen.
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)